src/uct/ib/base/ib_md.c

```diff
@@ -990,7 +1058,8 @@ static ucs_status_t uct_ib_mkey_pack(uct_md_h uct_md, uct_mem_h uct_memh,
      */
     if ((memh->flags & UCT_IB_MEM_ACCESS_REMOTE_ATOMIC) &&
         !(memh->flags & UCT_IB_MEM_FLAG_ATOMIC_MR) &&
-        (memh != &md->global_odp))
+        (memh != &md->global_odp) &&
+        md->config.enable_umr_pack)
```

Comment:
why do we have 'enable_umr_pack' in this PR?

Reply:
yep, duplication of existing functionality. removed